CSS/WorkCards.css

```diff
@@ -1,5 +1,5 @@
 .ArtData {
-  background-image: url(/images/Pgbg.svg);
+  background-image: url(../images/Pgbg.svg);
```

Comment:
- [ ] I see that you have fix the path issue for the images, but the deployed site is still using `url(/images/Pgbg.svg)`, check out what Dev Tools show on your site:

Dev tools:
![Captura de pantalla 2022-05-14 a la(s) 2 43 05 p  m](https://user-images.githubusercontent.com/83717285/168446161-a9ba2f35-6b45-443a-9cd1-1611782e869d.png)

This is causing the images not to be displayed, but I changed on Dev Tools the path adding `../` and it worked, so probably it just needs to be updated to your last version of the branch, try pushing again to make it deploy your last changes. 

Your deployment:
![Captura de pantalla 2022-05-14 a la(s) 2 39 06 p  m](https://user-images.githubusercontent.com/83717285/168446242-c408eb27-64c2-4854-8e4a-a9f67aa67c8f.png)

Reply:
@smunozmo thanks for reviewing my PR. Ya, it's because the GitHub page is directed to the main branch and this branch is still not merged in the main therefore it's showing like that. Now, what do you suggest to me in that?